docs/conf.py

```diff
@@ -506,6 +650,12 @@ def process_docstring(app, what, name, obj, options, lines):
 from legacy_redirect import build_legacy_redirect
 
 
+def strip_ipython_magic(app, docname, source):
```

Comment:
add a docstring comment just to clarify the function. should this be removing the whole line?

Reply:
Yep - IPython magic is not always one word (e.g. `%%writefile output.txt`) but it is always exactly one line.